idaes_examples/notebooks/docs/surrogates/SCO2_example/ALAMO/SCO2_alamo_surrogate.ipynb

```diff
@@ -0,0 +1,555 @@
+    "pressure and temperature of the system. The analytical equation of getting the enthalpy and entropy from pressure and temperature are in the differential form and would make the problem a DAE system. To counter this problem and keep the problem algebric, we will use the surrogates and relate enthalpy and entropy with the pressure and temperature as an algebric equation.\n",
+    "\n",
+    "### 1.2 Supercritical CO2 cycle process\n",
+    "\n",
+    "![image.png](attachment:image.png)\n",
+    "\n",
+    "The above flowsheet will be used to optimize the design for the cooling of the fusion reactor using supercritical CO2 cycle. We shall focus on training the surrogate for this notebook and move to constructing the flowsheet and the properties package in the subsequent notebooks. The take away from this flowsheet is that, 3 variables can be measured in any given unit which are flow, pressure and temperature. Thus, surrogate should have pressure and temperature as the inputs.\n",
+    "\n",
+    "In this example, we will train a model using AlamoTrainer for our data and then demonstrate that we can solve an optimization problem with that surrogate model. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Training and Validating Surrogate\n",
+    "\n",
+    "First, let's import the required Python and IDAES modules:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import statements\n",
+    "import os\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "# Import IDAES libraries\n",
+    "from idaes.core.surrogate.sampling.data_utils import split_training_validation\n",
+    "from idaes.core.surrogate.alamopy import AlamoTrainer, AlamoSurrogate\n",
+    "from idaes.core.surrogate.plotting.sm_plotter import (\n",
+    "    surrogate_scatter2D,\n",
+    "    surrogate_parity,\n",
+    "    surrogate_residual,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.1 Importing Training and Validation Datasets\n",
+    "\n",
+    "In this section, we read the dataset from the CSV file located in this directory. 500 data points were simulated for S-CO2 physical properties using REFPROP package. This example is trained on the entire dataset to have cover different ranges of pressure and temperature. The data is separated using an 80/20 split into training and validation data using the IDAES split_training_validation() method.\n",
+    "\n",
+    "We rename the column headers because they contained \".\", we change \".\" to \"_\" as ALAMO accepts alphanumerical characters or underscores as the labels for input/output. Further, the input variables are ***CO2SM_Pressure***, ***CO2SM_Temperature***, while the output variables are ***CO2SM_CO2_Enthalpy***, ***CO2SM_CO2_Entropy***, hence we slice them and create the input and output data. "
```

Comment:
Check your formatting here - the `***` is not rendered well in GitHub (and likely notebooks in general). `**` indicates bold font in Markdown, do this ends up being rendering the text in bold with a single `*` before and after the text.

Also, I am not sure if readers will understand what you mean by "slice" in this context - I think you probably need to be clearer.